changelog/next/breaking-changes/2999-replace-nil-with-null.md

```diff
@@ -0,0 +1,3 @@
+The non-value literal in expressions has a new syntax: `null` replaces its old
```

Comment:
The filename is missing a dash: It should be `2999--xxx.md`.

Reply:
Fixed dashes for two changelog files in https://github.com/tenzir/vast/pull/2999/commits/c7a69888d211895c004793352d852b0526a292f8